patches/chrome-browser-resources-history-synced_device_manager.html.ts.patch

```diff
@@ -1,20 +1,23 @@
 diff --git a/chrome/browser/resources/history/synced_device_manager.html.ts b/chrome/browser/resources/history/synced_device_manager.html.ts
-index e2676d4c40cb24854939d8d96f2296f14467aa7b..4446d5c253f8a8ba3af11ecd8a469ea41135e741 100644
+index e2676d4c40cb24854939d8d96f2296f14467aa7b..f072f0894fd9612fc1b3463c2e26bb5cd672838e 100644
 --- a/chrome/browser/resources/history/synced_device_manager.html.ts
 +++ b/chrome/browser/resources/history/synced_device_manager.html.ts
-@@ -31,6 +31,7 @@ export function getHtml(this: HistorySyncedDeviceManagerElement) {
+@@ -31,18 +31,6 @@ export function getHtml(this: HistorySyncedDeviceManagerElement) {
    ${this.noSyncedTabsMessage_()}
  </div>
  
-+<!--
- <div id="sign-in-guide"
-     ?hidden="${!this.showSignInGuide_()
-       || this.replaceSyncPromosWithSignInPromos_}">
-@@ -42,6 +43,7 @@ export function getHtml(this: HistorySyncedDeviceManagerElement) {
-     $i18n{turnOnSyncButton}
-   </cr-button>
- </div>
-+-->
- 
+-<div id="sign-in-guide"
```

Comment:
this should be commented out like the original patch

Reply:
Fixed here https://github.com/brave/brave-core/pull/32111/commits/3cd3bccb1dea63fbda8ddf15c0ba9d7c1ddec681.

Reply:
oh yeah, I think I flagged this one before because it should be hidden by default